framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/ClusteredWorkbenchKieServerPersistentScenarioImpl.java

```diff
@@ -168,4 +174,52 @@ public SsoDeployment getSsoDeployment() {
     public GitProvider getGitProvider() {
         return gitProvider;
     }
+
+    @Override
+    protected String overridesVersionTag() {
+        if (request.getUpgradeSettings() != null) {
+            return OpenShiftOperatorConstants.getKieOperatorUpgradeFromImageTag();
+        }
+
+        return null;
+    }
+
+    private void configureOperatorForUpgrades() {
+        if (request.getUpgradeSettings() != null) {
+            Upgrades upgrades = new Upgrades();
+            upgrades.setEnabled(true);
+            upgrades.setMinor(request.getUpgradeSettings().isMinor());
+            kieApp.getSpec().setUpgrades(upgrades);
```

Comment:
Why you don't configure kieApp in scenario builder instead of here?

Reply:
Good idea, will do.